01-Introducing-Molecular-Dynamics/01b-Rigid-Body-Molecular-Dynamics-Simulations.ipynb

```diff
@@ -0,0 +1,362 @@
+{
```

Comment:
<p>Save the identity array since you use it so frequently.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/54/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>